homeassistant/components/isy994/const.py

```diff
@@ -201,14 +213,6 @@
         ],  # Does a startswith() match; include the dot
         FILTER_ZWAVE_CAT: (["104", "112", "138"] + list(map(str, range(148, 180)))),
     },
-    Platform.BUTTON: {
```

Comment:
There aren't any devices that get sorted automatically into `buttons` or `numbers`, so these can be removed after breaking up `PLATFORMS` and `NODE_PLATFORMS`